src/container/srv_container.c

```diff
@@ -4649,8 +4696,12 @@ cont_op_with_svc(struct ds_pool_hdl *pool_hdl, struct cont_svc *svc,
 	rdb_tx_end(&tx);
 out:
 	/* Propagate new snapshot list by IV */
-	if (rc == 0 && (opc == CONT_SNAP_CREATE || opc == CONT_SNAP_DESTROY))
-		ds_cont_update_snap_iv(svc, in->ci_uuid);
+	if (rc == 0) {
+		if (opc == CONT_SNAP_CREATE || opc == CONT_SNAP_DESTROY)
+			ds_cont_update_snap_iv(svc, in->ci_uuid);
+		else if (opc == CONT_PROP_SET)
+			ds_cont_prop_iv_update(svc, in->ci_uuid);
+	}
```

Comment:
[Question] In addition to adding this new IV update call, does this patch also intend to delete any existing IV update calls? For instance, the one in set_prop?

Reply:
Ah, yes, that IV update in set_prop should be removed. But this will be conflicted with PR-11467, actually that patch change "clear status" process, let me merge the patch altogether.